packages/client/src/views/Performance/LeaderBoards.test.tsx

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
These test files can be removed from the PR I think as compiler should in most cases make sure that the initial render wouldn't cause an exception. The added reliability added by these tests doesn't overcome the added execution time of these tests

Reply:
if i am understanding i should remove `leaderboards.test.tsx`, `statistics.test.tsx`?

Reply:
That's correct and I'd even consider removing https://github.com/opencrvs/opencrvs-core/pull/4721/files#diff-4a8cefa45443494ebf4cfcabb22e2611429e22448dae6fc02b785753967d85d4 as the value it adds in it's current form is minimal